src/server/models/user.js

```diff
@@ -1,51 +1,43 @@
-// We need to use require instead of import
-// because this file is imported by sequelize-cli(without babel)
-const Sequelize = require('sequelize');
-const postgresConnection = require('../../../config/postgresConnection');
+import Sequelize from 'sequelize';
+import db from '../config/database-config';
 
-const { Model } = Sequelize;
+const defaultUserImg =
+  // eslint-disable-next-line max-len
+  'https://www.google.com/url?sa=i&source=images&cd=&ved=2ahUKEwjp7uSztJ3jAhWBjKQKHSZwAqkQjRx6BAgBEAQ&url=https%3A%2F%2Fdeathbattlefanon.fandom.com%2Fwiki%2FChin-Chin&psig=AOvVaw3O0n5ETwjexM5gl5avRJsb&ust=1562403407133663';
 
-const {
-  development: { username, password, database, host, dialect },
-} = postgresConnection;
-const sequelize = new Sequelize(database, username, password, {
-  host,
-  dialect,
-});
-
-const userSchema = {
-  id: {
-    allowNull: false,
-    autoIncrement: true,
-    primaryKey: true,
-    type: Sequelize.INTEGER,
-  },
-  name: {
-    type: Sequelize.STRING,
-  },
-  createdAt: {
-    allowNull: false,
-    type: Sequelize.DATE,
+const User = db.define(
+  'User',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
+      primaryKey: true,
+      unique: true,
+    },
+    googleId: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      unique: true,
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    name: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    img: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      defaultValue: defaultUserImg,
+    },
   },
-  updatedAt: {
-    allowNull: false,
-    type: Sequelize.DATE,
+  {
+    // eslint-disable-next-line max-len
+    timestamps: false, // Колонки createdAt и updatedAt будут созданы автоматически
```

Comment:
1. We need to have to createdAt and updatedAt to have a consistancy between model and existing database.
2. Avoid russian letters, because we need to keep our code extendable.